### PR TITLE
refactor(all): rename function, format RNKeys.gradle

### PR DESCRIPTION
**Pull request by MasonLe2497**
## Changelog

[refactor] [cpp] - Rename function getJniJsonStringyfyData -> getJniJsonStringifyData, format RNKeys.gradle code

### CODE REVIEW

File: android/RNKeys.gradle [hunk 1]
```diff
@@ -61,62 +61,66 @@ def loadKeys(){
...
 
     project.ext.set("keys", keys)
 
     android {
       defaultConfig {
-        keys.each { k, v ->
-          def escaped = v.replaceAll("%","\\\\u0025")
-          buildConfigField "String", k, "\"$v\""
-          resValue "string", k, "\"$escaped\""
+        keys.each {
```
Comment (numandev1) [inline android/RNKeys.gradle:106]: @MasonLe2497 can you reverse only this formatting on because it is unable to readable

**Review by numandev1 — CHANGES_REQUESTED** [review]

File: android/RNKeys.gradle (hunk 1 above)
Comment (MasonLe2497) [reply android/RNKeys.gradle]: Yep,  i updated my code. thanks

File: android/RNKeys.gradle [hunk 2]
```diff
@@ -12,8 +12,9 @@ def getCurrentFlavor() {
   def pattern = Pattern.compile("(?:.*:)*[a-z]+([A-Z][A-Za-z0-9]+)")
   def flavor = ""
 
-  gradle.getStartParameter().getTaskNames().any { name ->
-    Matcher matcher = pattern.matcher(name)
+  gradle.getStartParameter().getTaskNames().any {
+    name ->
```
Comment (numandev1) [inline android/RNKeys.gradle:16]: can you write name on line:15?

**Review by numandev1 — CHANGES_REQUESTED** [review]
@MasonLe2497 can you squash commit into one commit?

File: android/RNKeys.gradle (hunk 2 above)
Comment (MasonLe2497) [reply android/RNKeys.gradle:16]: yeap, got it

**Review by numandev1 — APPROVED** [review]
Thanks a lot ❤️

**Pull request merged**